src/components/molecules/Visualizer/Engine/ref.ts

```diff
@@ -22,6 +22,7 @@ export type EngineRef = {
 };
 
 export type SceneMode = "3d" | "2d" | "columbus";
+export type indicatorTypes = "default" | "crosshair" | "custom";
```

Comment:
export type IndicatorTypes = "default" | "crosshair" | "custom";

Please keep consistent with spacing, naming, etc. As you can see above all types are capitalized.

Reply:
Sorry it's my suggestion